iOS/MMIWG2S/ViewControllers/IntroViewController.swift

```diff
@@ -0,0 +1,206 @@
+        gradientView.frame = view.frame
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+        ])
+        
+        setupUI()
+    }
+    
+    
+    func setupUI() {
+    
+        //Setup UI
+        let labelMMIW = UILabel()
+        let labelMMIWInfo = UILabel()
+        let buttonNext = UIButton()
+ 
+        let spacerView1 = UIView()
+        let spacerView2 = UIView()
+
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        spacerView1.translatesAutoresizingMaskIntoConstraints = false
+
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        spacerView2.translatesAutoresizingMaskIntoConstraints = false
+
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+        view.addSubview(spacerView1)
+
+        let pageControlViews: [UIView] = [bar1, bar2, bar3, bar4]
+        
+        guard page < pageControlViews.count else {
+            print("page is outside pageControlViews ")
+            return
+        }
+        pageControlViews.forEach {
+            $0.backgroundColor = .white
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+
+        
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = String.slideviewheader
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont.roboto36
+
+        labelMMIWInfo.numberOfLines = 0
+        labelMMIWInfo.textColor = UIColor.white
+        labelMMIWInfo.font = UIFont.roboto25
+        labelMMIWInfo.text = String.slidetext(slidetext2:mmiwtext)
+        //initialize pcs
+
+        pageControlViews[page].backgroundColor = .red
+        
+        buttonNext.setTitleColor(UIColor.white, for:.normal)
+        buttonNext.setTitleColor(UIColor.green, for:.selected)
+
+        buttonNext.imageEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.contentEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.titleLabel?.font = UIFont.systemFont(ofSize: 17)
+        buttonNext.setTitle(String.next, for:.normal)
+        
+        buttonNext.addTarget(self, action: #selector(self.changePage(sender:)), for: .touchUpInside)
+        
+        let bottomView = UIView(frame:.zero)
+        bottomView.translatesAutoresizingMaskIntoConstraints = false
+        pageControlViews.forEach { bottomView.addSubview($0)
+        }
```

Comment:
for this we'll want either the `}` to be inline with the `{` or for the `{`, the content, and the `}` to be on separate lines. (i.e. either all in one line or as three separate lines)